packages/server/src/queue/queue-clean/queue-clean.service.spec.ts

```diff
@@ -75,87 +46,33 @@ describe('QueueService', () => {
       const question = await QuestionModel.findOne({});
       expect(question.status).toEqual('Queued');
     });
-
-    it('queue gets cleaned when force parameter is passed, even with staff present', async () => {
-      const ta = await UserFactory.create();
-      const queue = await QueueFactory.create({ staffList: [ta] });
+    it('if no staff are present all questions with open status are marked as stale', async () => {
+      const queue = await QueueFactory.create({});
       const question = await QuestionFactory.create({
         status: OpenQuestionStatus.Queued,
         queue: queue,
       });
 
-      await service.cleanQueue(queue.id, true);
-
+      await service.cleanQueue(queue.id);
       await question.reload();
       expect(question.status).toEqual('Stale');
     });
-
-    it('if no staff are present all questions with open status are marked as stale', async () => {
-      const ofs = await ClosedOfficeHourFactory.create();
-      const queue = await QueueFactory.create({ officeHours: [ofs] });
+    it('queue gets cleaned when force parameter is passed, even with staff present', async () => {
+      const ta = await UserFactory.create();
+      const queue = await QueueFactory.create({ staffList: [ta] });
       const question = await QuestionFactory.create({
         status: OpenQuestionStatus.Queued,
         queue: queue,
       });
 
-      await service.cleanQueue(queue.id);
+      await service.cleanQueue(queue.id, true);
+
       await question.reload();
       expect(question.status).toEqual('Stale');
     });
 
-    it('checkout all staff from all queues', async () => {
```

Comment:
I think this test probably should still remain as well?

Reply:
yep